catalog/core/catalog-core-commons/src/main/java/ddf/util/XSLTUtil.java

```diff
@@ -32,12 +33,24 @@
 import org.w3c.dom.Document;
 
 public final class XSLTUtil {
-    private static final DocumentBuilderFactory DBF =
-            DocumentBuilderFactory.newInstance(org.apache.xerces.jaxp.DocumentBuilderFactoryImpl.class.getName(),
-                    org.apache.xerces.jaxp.DocumentBuilderFactoryImpl.class.getClassLoader());
-
     private static final Logger LOGGER = LoggerFactory.getLogger(XSLTUtil.class);
 
+    private static final DocumentBuilderFactory DBF;
+
+    static {
+        DBF = DocumentBuilderFactory.newInstance(
+                org.apache.xerces.jaxp.DocumentBuilderFactoryImpl.class.getName(),
+                org.apache.xerces.jaxp.DocumentBuilderFactoryImpl.class.getClassLoader());
+        try {
+            DBF.setFeature("http://apache.org/xml/features/nonvalidating/load-dtd-grammar", false);
```

Comment:
The one downside to putting all of them in a common location is that it now imposes a dependency on downstream projects that they may not want or break working impls for no real reason.